app/js/components/manageApps/Addon.jsx

```diff
@@ -208,7 +208,7 @@ class Addon extends Component {
         });
     } else {
       const applicationDistribution = location.href.split('/')[3];
-      axios.get(`/${applicationDistribution}/module/owa/deleteApp.form?appName=${name}`)
+      axios.get(`/${applicationDistribution}/module/owa/deleteApp.form?appName=${name}&requester=addonmanager`)
```

Comment:
The server should not be aware of addonmanager to make special actions for it. I would rather use a parameter name like returnUrl

Reply:
Hi @dkayiwa. I've done what you said and change the parameter from `requester` to `returnURL` and I'm returning whatever is passed in the controller. The OWA part worked but it's failing here https://modules-refapp.openmrs.org/openmrs/module/owa/manage.form. I guess it's because the `returnURL` parameter is not included in the jsp file. How do I go about it?

Reply:
When the returnUrl parameter is not passed in, you do the default. Else, it is what you return.
So the addon manager does something like @returnUrl=manager.form

Does this make sense?

Reply:
@dkayiwa Yes, it does make sense now. I have also updated the PR